src/odemis/driver/xt_client.py

```diff
@@ -2106,7 +2108,7 @@ def _updateSettings(self):
         super(MultiBeamScanner, self)._updateSettings()
         # Polling XTtoolkit settings
         try:
-            pitch = self.parent.get_pitch() * 1e-6
+            pitch = self.parent.get_pitch()
```

Comment:
That doubly doubty!
The multiplication factor should stay, and the function name should change.